### ISSUE TITLE
64 bit version of fife dont build on appveyor

### ISSUE THREAD

**Issue by LinuxDonald** [fifengine/fifengine#975]
I need help on this.
Log i attached
[log.txt](https://github.com/fifengine/fifengine/files/737557/log.txt)

**Comment by Niektory** [comment]
It seems AppVeyor is building 32-bit binaries even in the x64 job. So after changing Python to 64-bit it fails because everything else is 32-bit.

**Comment by Niektory** [comment]
I think `-A "x64"` needs to be passed to cmake to build for x64.

**Comment by jakoch** [comment]
Well, you can't simply rename x86 to x64 and hope for a x64 build.
You are building with the x86 toolchain, which produces an x86/32bit builds.
You need to configure a toolchain+compiler for x64.
Add a new compiler toolchain to the travis and appveyor matrix. 
E.g. so that you use `cmake .. -G "Visual Studio 14 Win64"`, `%generator` = add a new generator to Appveyor.
Keep in mind, that all dependencies have to be build for that bitsize, too (superbuild), before fifengine can link against them. You can't link x64 against x86 prebuilds, etc..

(Sidenote: The number of build paths and build artifacts increases (`^2`).  3 os x 2 bitsizes x 3 compilers = x artifacts. Then the installers: 2x2. And all of that with only 1 release engineer, because cloning failed ;)

**Comment by LinuxDonald** [comment]
Yeah i should fork it an work on it. When its done i should create an pr :)
